### ISSUE TITLE
feat: add test coverage for TEE Plugin

### ISSUE THREAD

**Issue by HashWarlock** [elizaOS/eliza#1324]
**Is your feature request related to a problem? Please describe.**
Currently there is not basic test coverage for the TEE Plugin.

**Describe the solution you'd like**
Solution for a github action will go as follows:
- Start published docker image for TEE Simulator
- Build and deploy docker image of eliza agent with `TEE_MODE=DOCKER`
- Generate a message to eliza
- Expect a derived key with an evm and solana wallet generated from a `WALLET_SECRET_SALT`
- Expect a RA Quote generation for the agent ID, Solana Wallet and EVM Wallet derived from the secret salt

**Describe alternatives you've considered**
None at the moment. Need a basic test first.

**Additional context**

**Comment by Luca-Blight** [comment]
Will you be working on this?

**Comment by HashWarlock** [comment]
Yeah, haven't submitted PR yet. Are you looking to write some tests as well?

**Comment by Luca-Blight** [comment]
Yes, but don't want to step on your toes hence why I asked. 
I'm new to TEEs so working on stuff like this can be a great way to learn.

**Comment by HashWarlock** [comment]
I would love some help 🙏 I've been so overloaded lately & I'm having a hard time managing. I can definitely review when you submit PR.

**Comment by Luca-Blight** [comment]
Cool! I will explore this further tomorrow and let you know if any questions.

**Comment by HashWarlock** [comment]
Closing. Added this to TEE Plugin awhile back here https://github.com/elizaOS/eliza/tree/main/packages/plugin-tee/__tests__
